lib/mongoid/association/macros.rb

```diff
@@ -36,10 +36,15 @@ module Macros
         # @api private
         class_attribute :aliased_associations
 
+        # @return [ Set<String> ] The set of associations that are configured 
+        #   with :store_as parameter.
```

Comment:
Should add additional context which explains why this is needed and reference the ticket.